hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -1071,9 +1072,10 @@ protected void loadAllInternal(boolean replaceExistingValues, Collection<Data> d
         MapEntrySetCodec.ResponseParameters resultParameters = MapEntrySetCodec.decodeResponse(response);
 
         InflatableSet.Builder<Entry<K, V>> setBuilder = InflatableSet.newBuilder(resultParameters.response.size());
-        SerializationService serializationService = getContext().getSerializationService();
+        InternalSerializationService serializationService
+                = ((InternalSerializationService) getContext().getSerializationService());
         for (Entry<Data, Data> row : resultParameters.response) {
-            LazyMapEntry entry = new LazyMapEntry(row.getKey(), row.getValue(), serializationService);
+            LazyMapEntry entry = new LazyMapEntry(row.getKey(), row.getValue(), serializationService, null);
             setBuilder.add(entry);
```

Comment:
it is not doing anything that requires extractors so passing null should be fine or a new constructor can be introduced.